moztelemetry/filter_service.py

```diff
@@ -28,10 +28,11 @@
 
 
 class SDB:
-    def __init__(self, prefix, months_retention=12, read_only=True):
+    def __init__(self, prefix, months_retention=12, read_only=True, s3_prefix=None):
```

Comment:
Side note: I think the plan is to provide the retention period for each data set in the sources.json metadata (but it isn't there now). `months_retention` could probably then be derived from that value.